README.md

```diff
@@ -17,14 +17,38 @@ To report a bug report or suggest a feature, please read our
 
 ## Getting started
 
-### Binary download
+### Setting up fpm
+
+#### Binary download
 `x86-64` binaries are available [to download](https://github.com/fortran-lang/fpm/releases) for Windows, MacOS and Linux.
 
 __Note:__ On Linux and MacOS, you will need to enable executable permission before you can use the binary.
 
 _e.g._ `$ chmod u+x fpm-v0.1.0-linux-x86_64`
 
-__Github actions:__ to setup *fpm* within Github actions for automated testing, you can use the [fortran-lang/setup-fpm](https://github.com/marketplace/actions/setup-fpm) action.
+#### Conda
+
+If you haven't yet, first enable the conda-forge channel:
+
+```
+conda config --add channels conda-forge
+```
+
+Fpm can be installed with:
+
+```
+conda create -n fpm fpm
+conda activate fpm
+```
+
+Conda can be installed from [miniconda](https://docs.conda.io/en/latest/miniconda.html)
+or from [miniforge](https://github.com/conda-forge/miniforge/releases).
```

Comment:
Let's recommend miniforge over miniconda, since we are distributing via conda-forge:

```suggestion
Conda can be installed from [miniforge](https://github.com/conda-forge/miniforge/releases)
or from [miniconda](https://docs.conda.io/en/latest/miniconda.html).
```